<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetupToolset</class>
 <widget class="QWidget" name="SetupToolset">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>355</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>398</width>
    <height>355</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>398</width>
    <height>355</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Looking Glass - Choose Toolset</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>.:/icons/LookingGlass.png</normaloff>.:/icons/LookingGlass.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../resources/resources.qrc">:/images/SuiteSplash.png</pixmap>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Profile:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="profileBox"/>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Sonde and Bathymetry Toolset</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
      <item>
       <widget class="QPushButton" name="runSondeToolsetButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Run Sonde and Bathymetry Toolset</string>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Glacier View Toolset</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0">
      <item>
       <widget class="QPushButton" name="runGlacierToolsetButton">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>32</height>
         </size>
        </property>
        <property name="text">
         <string>Run Glacier View Toolset</string>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
  <include location="../resources/resources.qrc"/>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
